README.md

```diff
@@ -1,19 +1,29 @@
 
-The openid-client-helper package allows for performing a single [OAuth 2.0](https://tools.ietf.org/html/rfc6749) authorization grant and maintaining separate authorization states for multiple resources protected by the same [Authorization Server](https://tools.ietf.org/html/rfc6749#section-1.1).
+The openid-client-helper package allows for performing a single OAuth 2.0 authorization grant and maintaining separate authorization states for multiple resources protected by the same [Authorization Server](https://tools.ietf.org/html/rfc6749#section-1.1).
 
 The authorization can be performed with the helper express middleware or with external means, like the Passport.js strategy. If the resulting token set contains refresh token, it can be used to obtain access tokens for individual resources. The helper exposes the redefined [node-fetch](https://www.npmjs.com/package/node-fetch#difference-from-client-side-fetch) `fetch` method for making HTTP requests to the resources protected by OAuth 2.0. Each individual request will be crafted with an access token specific to the requested resource. The token will be automatically obtained and, if necessary, renewed while handling the request.
 
 This means that after initial authorization by the [Resource Owner](https://tools.ietf.org/html/rfc6749#section-1.1) (RO), the RP will be able to perform API requests with [Audience Restricted Access Tokens](https://tools.ietf.org/html/draft-ietf-oauth-security-topics-13#section-4.8.1.3) according to the best current practices (BCPs), performing authorization for individual resources entirely via the back-channel; that is, without further RO involvement.
 
 ## Features
 
+* Authorization and authentication middleware
+* Deauthorization and RP initiated logout middleware
+* Audience restricted access tokens
+* Access tokens are automatically refreshed and included in requests made to protected resources
+* Easy setup for multiple OIDC providers
+* Based on openid-client library
+* Can work alongside Express and Passport
+
+## How it works
+
 An instance of the openid-client-helper is provides aims to aid with a quick implementation of the entire authentication and authorization flow with a set of [Express](https://expressjs.com/) middleware and public methods built on top of the openid-client functionality. The helper accepts configuration parameters for a combination of the OIDC Provider (OP) and the RP. Multiple combinations may be used in the same Node.js application simultaneously. Then, the Authorization Code grant can be performed by calling the helper's instance middleware in route handlers. The middleware optionally accepts additional authorization parameters not originally provided in the helper configuration.
```

Comment:
Wondering which verb we'd like to use here? We've got is, provides, and aims. Wondering what you think of this: 
An instance of the he openid-client-helper provides a quick implementation...